docs/content/integrations/dbt/reference.mdx

```diff
@@ -537,20 +537,37 @@ Note that Dagster allows the optional specification of a [`code_version`](/conce
 
 ## Loading dbt tests as asset checks
 
-<Note>
-  <strong>Dagster asset checks are currently an experimental feature. </strong>{" "}
-  To provide feedback, join our{" "}
-  <a href="https://github.com/dagster-io/dagster/discussions/16527">
-    GitHub discussion
```

Comment:
We should keep the note but change the contents.

1. Asset checks for dbt have been enabled by default, starting 1.7.
2. You need at least `dbt-core>=1.6` to use its full functionality.

We can remove the second bullet once `dbt-core==1.5` is deprecated at the end of April 2024.